docs/crashstorage/elasticsearch.rst

```diff
@@ -127,41 +127,86 @@ contains data about each field from Elasticsearch that can be manipulated.
-|                      | filers of this field in Super Search.                   |
-+----------------------+---------------------------------------------------------+
-| permissions_needed   | Permissions needed for a user to access this field.     |
-+----------------------+---------------------------------------------------------+
-| form_field_choices   | Choices offered for filters of that field in the        |
-|                      | Super Search form.                                      |
-+----------------------+---------------------------------------------------------+
-| is_exposed           | Is this field exposed as a filter?                      |
-+----------------------+---------------------------------------------------------+
-| is_returned          | Is this field returned in results?                      |
-+----------------------+---------------------------------------------------------+
-| has_full_version     | Does this field have a full version in Elasticsearch?   |
-|                      | Enable only if you use a multitype field in the         |
-|                      | storage mapping.                                        |
-+----------------------+---------------------------------------------------------+
-| storage_mapping      | Mapping that is used in Elasticsearch for this field.   |
-|                      | See Elasticsearch documentation for more info.          |
-+----------------------+---------------------------------------------------------+
+**name**
+    The name of the field.
+
+**description**
+    Brief description of the field.
+
+    This shows up in the `Super Search API documentation
+    <https://crash-stats.mozilla.com/documentation/supersearch/api/>`_.
+
+**namespace**
+    The dotted name space for the source of the value of this field.
+
+    Examples:
+
+    * ``raw_crash``
+    * ``processed_crash``
+    * ``processed_crash.json_dump.crashing_thread``
+
+**in_database_name**
+    This is the name used to store this field value in Elasticsearch and other
+    places.
+
+**query_type**
+    Specifies the operators that can be used with this field in Super Search.
+    See the list of query types below.
+
+**data_validation_type**
+    Specifies how values are validated when passed to filters of this field
+    in Super Search.
+
+    Possible values: ``str``, ``enum``, ``bool``, ``int``, ``datetime``,
+
+**permissions_needed**
+    Permissions needed for a user to access this field.
+
+**form_field_choices**
+    Possible values for this field in the Super Search form.
+
+**is_exposed**
+    Is this field exposed as a filter?
+
+**is_returned**
+    Is this field returned in results?
+
+**has_full_version**
+    Does this field have a full version in Elasticsearch? Enable only if you use
+    a multitype field in the storage mapping.
+
+**storage_mapping**
+    Mapping that is used in Elasticsearch for this field. See below for more
+    information.
+
+
+We also have these properties:
```

Comment:
So we have to include these but always set them to the same value? It might be good to explicitly say that as part of this demarcation rather than in the descriptions below.

Reply:
That makes sense to me!